README.md

```diff
@@ -1 +1,46 @@
-##Network Service
+# Network Service 
+
+Network layer for creating different set of network requests like GET, POST, PUT, DELETE 
+⚠️ The network layer is under active development.
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) | | 
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) | |
+| ![Swift](https://img.shields.io/badge/Swift-5.8-orange) | |
+| ![SDKs for iOS](https://img.shields.io/badge/SDKs%20for%20iOS-15%2B-lightgrey) | |
+| ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-yellow) | [Link](https://github.com/nicklockwood/SwiftFormat) |
+| ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)| [Link](https://github.com/realm/SwiftLint) |
+
+## Table of contents
+* [General info](#general-info)
```

Comment:
nit:
Why does it appear in the middle of the page, not at the top?
Do we need it, if we have default  navigation for readme?
<img width="896" alt="Screenshot 2023-06-15 at 13 46 23" src="https://github.com/ios-course/simple-network-service/assets/32869814/e200def1-bfcf-4266-8c75-4156636c9dfd">

Reply:
fixed